src/displayapp/screens/Dice.cpp

```diff
@@ -14,9 +14,8 @@ static void btnRollEventHandler(lv_obj_t* obj, lv_event_t event) {
   }
 }
 
-Dice::Dice(DisplayApp* app, Controllers::DateTime& dateTime) : Screen(app) {
-
-  srand(dateTime.Uptime().count() * time(nullptr));
+Dice::Dice(DisplayApp* app, System::SystemTask& systemTask) : Screen(app), systemTask {systemTask} {
```

Comment:
SystemTask is never used, is it?

Reply:
You are correct; for some reason I thought at one point that I would need it to get the system tick count, which is now being used for the seed (working better than uptime and other things I tried-- no operations needed, and seems there's about 1 tick per ~0.1 ms so very low chance of seed collision!) -- I must have mistakenly left in the argument & reference after that. Thanks for catching that! Reference removed in adc97a03ba0b9555970692ddf01c7e9ce56e2e20